src/App.tsx

```diff
@@ -4,6 +4,10 @@ import Carousel from './components/Carousel';
 
 interface State {
   images: string[];
+  step: string,
+  frameSize: string,
+  itemWidth: string,
+  animationDuration: string,
```

Comment:
```suggestion
  step: number,
  frameSize: number,
  itemWidth: number,
  animationDuration: number,
```
It will be more clear